src/main/java/uk/gov/hmcts/reform/hmc/api/Application.java

```diff
@@ -21,6 +22,7 @@
         "HideUtilityClassConstructor") // Spring needs a constructor, its not a utility class
 @SpringBootConfiguration
 @EnableAutoConfiguration
+@EnableAsync
```

Comment:
is this needed?

Reply:
Removed Yogendra .. apologies initially removed only in service class but forgot to remove here Thank you